cypress/e2e/rooms_administration/externalRoomVisibleOnStagingEnv.feature

```diff
@@ -0,0 +1,97 @@
+        Given I am logged in as a '<admin>' at '<namespace>'
+        When I click on administration in menu
+        When I navigate to rooms administration page via the submenu
+        Then I see the rooms administration page
+        Then I see the room '<room_name>' on the rooms administration page
+        Then I see the room '<room_name>' has correct internal members count '<internal_members_count>'
+        Then I see the room '<room_name>' has correct external members count '<external_members_count>'
+        Then I see the room '<room_name>' has correct total members count '<total_members_count>'
+
+        # admin (Felix Mendelssohn-Gymnasium) can access member management for room and see external and internal user
+        When I click on three dot menu in the room admin page for room '<room_name>'
+        When I click on manage room members in the three dot menu
+        Then I see the admin page Edit participants of room '<room_name>'
+        Then I see '<participant_external_name>' in the admin room participants list
+        Then I see '<participant_origin_name>' in the admin room participants list
+
+        # external teacher (Goethe-Gymnasium) deletes the room and can not see it anymore
+        Given I am logged in as a '<teacherExt>' at '<namespace>'
+        When I go to rooms overview
+        When I go to room '<room_name>'
+        Then I see the detail page of room '<room_name>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'delete'
+        Then I see confirmation modal for deleting the room
+        When I click on delete button in confirmation modal
+        Then I do not see '<room_name>' on room overview page
+
+        # admin (Felix Mendelssohn-Gymnasium) can not see the room anymore
+        Given I am logged in as a '<admin>' at '<namespace>'
+        When I click on administration in menu
+        When I navigate to rooms administration page via the submenu
+        Then I see the rooms administration page
+        Then I do not see the room '<room_name>' on the rooms administration page
+
+        # origin teacher (Felix Mendelssohn-Gymnasium) can not see the room anymore
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to rooms overview
+        Then I do not see '<room_name>' on room overview page
+
+        @staging_test
+        Examples:
+            | admin      | teacher      | teacherExt      | namespace | room_name         | participant_origin_school   | role_name      | participant_origin_name  | participant_external_name  | internal_members_count | external_members_count | total_members_count |
+            | admin1_brb | teacher1_brb | teacherExt1_brb | brb       | Cypress Room Name | Felix Mendelssohn-Gymnasium | Lernbegleitend | XYZ                      | Carlo                      | 1                      | 1                      | 2                   |
```

Comment:
same here, what does it mean by `XYZ`?
<img width="1339" height="763" alt="Screenshot 2025-09-04 at 00 12 17" src="https://github.com/user-attachments/assets/e30fdde6-c5ca-4fe6-911c-d477dc33d1a3" />